### ISSUE TITLE
Start recording a little bit deferred

### ISSUE THREAD

**Issue by tsdh** [Malabarba/camcorder.el#4]
I've just made my first screencast in https://tsdh.wordpress.com/2015/09/17/improving-avy-goto-char-timer/.  As you can see, in the beginning of the screencast you can still see the minibuffer retracting (from `M-x camcorder-mode RET` via `ivy-mode`s completion) and messages about something with OGV.  I'd suggest to start recording only after some short delay in order to exclude those artifacts from the screencast.

**Comment by Malabarba** [comment]
That's a good suggestion. I'll do it soon. Should be as simple as adding something like `sleep 1` to the command.

**Comment by tsdh** [comment]
Great, thanks!
